From ce0aa65799c66d89da82d5c096f1cd0d6dd6ec39 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Mon, 19 Mar 2018 14:52:34 +0800
Subject: [PATCH 0661/1200] arm64: dts: a70x0: a80x0: Correct spi flash
 compatible string

SPI flash's compatible strings should include "spi-flash", this patch
adds "spi-flash" to SPI flash compatible strings for armada 7k/8k and
8k plus boards.

Change-Id: Iec209f1b07a0d09c776b168a35c44356bc419ad4
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/55227
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
---
 arch/arm/dts/armada-7040-db-nand.dts | 2 +-
 arch/arm/dts/armada-7040-db.dts      | 2 +-
 arch/arm/dts/armada-8040-db.dts      | 2 +-
 arch/arm/dts/armada-8040-mcbin.dts   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/dts/armada-7040-db-nand.dts b/arch/arm/dts/armada-7040-db-nand.dts
index ed9eb0a79b..e3a5a715c0 100644
--- a/arch/arm/dts/armada-7040-db-nand.dts
+++ b/arch/arm/dts/armada-7040-db-nand.dts
@@ -130,7 +130,7 @@
 	spi-flash@0 {
 		#address-cells = <0x1>;
 		#size-cells = <0x1>;
-		compatible = "jedec,spi-nor";
+		compatible = "jedec,spi-nor", "spi-flash";
 		reg = <0x0>;
 		spi-max-frequency = <20000000>;
 
diff --git a/arch/arm/dts/armada-7040-db.dts b/arch/arm/dts/armada-7040-db.dts
index b76014a503..1ed23ecc55 100644
--- a/arch/arm/dts/armada-7040-db.dts
+++ b/arch/arm/dts/armada-7040-db.dts
@@ -123,7 +123,7 @@
 	spi-flash@0 {
 		#address-cells = <0x1>;
 		#size-cells = <0x1>;
-		compatible = "jedec,spi-nor";
+		compatible = "jedec,spi-nor", "spi-flash";
 		reg = <0x0>;
 		spi-max-frequency = <20000000>;
 
diff --git a/arch/arm/dts/armada-8040-db.dts b/arch/arm/dts/armada-8040-db.dts
index ddcbfc4901..35bddca06e 100644
--- a/arch/arm/dts/armada-8040-db.dts
+++ b/arch/arm/dts/armada-8040-db.dts
@@ -252,7 +252,7 @@
 	spi-flash@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "jedec,spi-nor";
+		compatible = "jedec,spi-nor", "spi-flash";
 		reg = <0>;
 		spi-max-frequency = <10000000>;
 
diff --git a/arch/arm/dts/armada-8040-mcbin.dts b/arch/arm/dts/armada-8040-mcbin.dts
index 31698c3fbe..880e76d02c 100644
--- a/arch/arm/dts/armada-8040-mcbin.dts
+++ b/arch/arm/dts/armada-8040-mcbin.dts
@@ -240,7 +240,7 @@
 	spi-flash@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "jedec,spi-nor";
+		compatible = "jedec,spi-nor", "spi-flash";
 		reg = <0>;
 		spi-max-frequency = <10000000>;
 
-- 
2.22.0

